x-pack/plugins/ml/server/routes/annotations.ts

```diff
@@ -103,10 +103,10 @@ export function annotationRoutes(
         const { indexAnnotation } = annotationServiceProvider(context);
 
         const currentUser =
-          securityPlugin !== undefined && securityPlugin.authc.getCurrentUser(request);
-        const user = currentUser ? currentUser : {};
+          securityPlugin !== undefined ? securityPlugin.authc.getCurrentUser(request) : {};
         // @ts-ignore username doesn't exist on {}
```

Comment:
looks like this `@ts-ignore` can go now

Reply:
That warning still shows up since `currentUser` can still be {} 😞

Reply:
ah ok. would it be worth changing the line above to make `currentUser` `null` or `undefined` in that situation?
As it would be caught by `currentUser?.username ??` on the line below.